rpc/src/rpc.rs

```diff
@@ -115,6 +115,22 @@ type RpcCustomResult<T> = std::result::Result<T, RpcCustomError>;
 pub const MAX_REQUEST_BODY_SIZE: usize = 50 * (1 << 10); // 50kB
 pub const PERFORMANCE_SAMPLES_LIMIT: usize = 720;
 
+fn rpc_account_index_from_account_index(account_index: &AccountIndex) -> RpcAccountIndex {
+    match account_index {
+        AccountIndex::ProgramId => RpcAccountIndex::ProgramId,
+        AccountIndex::SplTokenOwner => RpcAccountIndex::SplTokenOwner,
+        AccountIndex::SplTokenMint => RpcAccountIndex::SplTokenMint,
+    }
+}
+
+fn _account_index_from_rpc_account_index(rpc_account_index: &RpcAccountIndex) -> AccountIndex {
+    match rpc_account_index {
+        RpcAccountIndex::ProgramId => AccountIndex::ProgramId,
+        RpcAccountIndex::SplTokenOwner => AccountIndex::SplTokenOwner,
+        RpcAccountIndex::SplTokenMint => AccountIndex::SplTokenMint,
+    }
+}
```

Comment:
Is this used anywhere?

Reply:
No hence the "_" but didn't wanna provide one way without the other.